src/glue_launcher_lambda/glue_launcher.py

```diff
@@ -236,12 +236,19 @@ def get_parameters():
             "MANIFEST_S3_INPUT_PARQUET_LOCATION_MISMATCHED_TIMESTAMPS"
         ]
 
-    if {"MANIFEST_S3_BUCKET", "MANIFEST_S3_PREFIX"}.issubset(os.environ):
-        s3_bucket = os.environ["MANIFEST_S3_BUCKET"]
-        s3_prefix = os.environ["MANIFEST_S3_PREFIX"]
-        _args.manifest_s3_output_location = f"s3://{s3_bucket}/{s3_prefix}"
-        _args.manifest_s3_bucket = s3_bucket
-        _args.manifest_s3_prefix = s3_prefix
+    if "MANIFEST_S3_OUTPUT_LOCATION" in os.environ:
+        _args.manifest_s3_output_location = os.environ["MANIFEST_S3_OUTPUT_LOCATION"]
+
+    if "MANIFEST_S3_BUCKET" in os.environ:
+        _args.manifest_s3_bucket = os.environ["MANIFEST_S3_BUCKET"]
+
+    if "MANIFEST_S3_PREFIX" in os.environ:
+        _args.manifest_s3_prefix = os.environ["MANIFEST_S3_PREFIX"]
+
+    if "MANIFEST_DELETION_PREFIXES" in os.environ:
+        _args.manifest_deletion_prefixes = os.environ[
+            "MANIFEST_DELETION_PREFIXES"
+        ].split(",")
```

Comment:
```suggestion
    if "MANIFEST_DELETION_PREFIXES" in os.environ:
        _args.manifest_deletion_prefixes = os.environ[
            "MANIFEST_DELETION_PREFIXES"
        ].remove(" ","").split(",")
```
In case env vars are passed in with or without spaces between comma.